arm-servermanagement/2016-07-01-preview/servermanagement.json

```diff
@@ -0,0 +1,2234 @@
+                        "schema": {
+                            "$ref": "#/definitions/Error"
+                        }
+                    }
+                },
+                "x-ms-long-running-operation": true
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServerManagement/nodes/{nodeName}/sessions/{session}/features/powerShellConsole/pssessions/{pssession}/tab": {
+            "post": {
+                "tags": [
+                    "PowerShell"
+                ],
+                "operationId": "PowerShell_TabCompletion",
+                "description": "gets tab completion values for a command.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The resource group name uniquely identifies the resource group within the user subscriptionId.",
+                        "minLength": 3,
+                        "pattern": "[a-zA-Z0-9]+"
+                    },
+                    {
+                        "name": "nodeName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The node name (256 characters maximum).",
+                        "maxLength": 256,
+                        "minLength": 1,
+                        "pattern": "^[a-zA-Z0-9][a-zA-Z0-9_.-]*$"
+                    },
+                    {
+                        "name": "session",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The sessionId from the user"
+                    },
+                    {
+                        "name": "pssession",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The PowerShell sessionId from the user"
+                    },
+                    {
+                        "name": "PowerShellTabCompletionParamters",
+                        "x-ms-client-flatten": true,
+                        "description": "Parameters supplied to the tab completion call.",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/
```

Comment:
Did you get the Visual Studio Code linter working ? The missing descriptions should have been caught by the editor.